README.md

```diff
@@ -217,153 +217,194 @@ varsig-body = *OCTET; Zero or more segments required by the kind of varsig (e.g.
 
 For example, here is an EdDSA signature for some content encoded as DAG-PB:
 
-`0x34ed01ae3784f03f9ee1163382fa6efa73b0c31ecf58c899c836709303ba4621d1e6df20e09aaa568914290b7ea124f5b38e70b9b69c7de0d216880eac885edd41c302`
+`%x34ed01ae3784f03f9ee1163382fa6efa73b0c31ecf58c899c836709303ba4621d1e6df20e09aaa568914290b7ea124f5b38e70b9b69c7de0d216880eac885edd41c302`
 
 ### 3.1 Varsig Prefix
 
-The varsig prefix MUST be `0x34`.
+The varsig prefix MUST be `%x34`.
 
 ### 3.2 Signature Header
 
 The prefix of the signature algorithm. This is often the [multicodec] of the associated public key, but MAY be unique for the signature type. The code MAY live outside the multicodec table. This field MUST act as a discriminant for how many expected fields come in the varsig body, and what each of them mean.
 
 ### 3.3 Varsig Body
 
-The varsig body MUST consist of zero or more segments, and MUST be defined by the signature algorithm.
+The varsig body MUST consist of one or more segments, and MUST be defined by the signature algorithm.
 
 Some examples include:
 
 * Raw signature bytes only
 * CID of [DKIM] certification transparency record, and raw signature bytes
-* Hash algorithm multicodec prefix, signature counter, HMAC, and raw signature bytes
+* Hash algorithm multicodec prefix, data encoding prefix, signature counter, nonce, HMAC, and raw signature bytes
 
-# 4 Registry of Common Signature Algorithms
+# 4 Payload Encoding
+
+The [IPLD] data model is encoding agnostic by design. This is very convenient in many applications, such as making for very convenient conversions between types for transmission versus encoding. Unfortunately signatures require signing over specific bytes, and thus over a specific encoding of the data.
+
+To facilitate this, the type `encoding-info` MAY be used:
+
+``` abnf
+encoding-info
+  = %x55   ; Raw bytes multicodec prefix
```

Comment:
Raw doens't make sense to me here because afaik there's no way to go from an IPLD object to Raw. By definition raw doesn't have any encode/decode functions.

Reply:
You're right that it doesn't have an encode/decode function. I would assume that we should have some ability to sign over a bare sequence of bytes outside of an object, no?

Reply:
I guess e.g. DAG-CBOR does have raw bytes, so that may be sufficient. Okay, removing!

Reply:
Actually scratch that. Even CBOR has a length header on binary. This inclusion makes it impossible express signatrues that are not diretcly in IPLD and need to interoperate with other systems.

How would you feel if `0x55` was a special case for when there was a single sibling, which unwraps it and passes it as raw bytes to the signature?

Reply:
I think that would be useful, but I'm not sure RAW is the right way to express this? I don't feel super strongly though.

Reply:
Okay sure, we can pick another tag. How about `0x5F`? It's `_` in ASCII, and there's no namespace collision the multicodec table.

Reply:
Sure!